eks/aws-node-termination-handler/kubernetes/PrometheusRules.dhall

```diff
@@ -0,0 +1,13 @@
+let imports = ../../../imports.dhall
+
+let Kubernetes = imports.Kubernetes
+
+let Prometheus = imports.Prometheus
+
+let Common = ./Common.dhall
+
+in  λ(common : Common.Type) →
+      Prometheus.PrometheusRule::{
+      , metadata = common.metadata ⫽ { labels = Some common.labels.prometheus }
+      , spec = Prometheus.PrometheusRuleSpec::{=}
+      }
```

Comment:
What's the point of including an empty rule set here?

Reply:
Once we will define the relevant rules it will be added